packages/autocannon-ui-frontend/src/components/RunOptionsForm.jsx

```diff
@@ -0,0 +1,290 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import {
+  Grid,
+  Container,
+  TextField,
+  Tooltip,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  Box,
+  Button
+} from '@material-ui/core'
+import T from 'prop-types'
+import HelpIcon from '@material-ui/icons/Help'
+
+import ProgressBar from './ProgressBar.jsx'
+import logo from '../assets/autocannon-banner.png'
+import { Alert } from '@material-ui/lab'
+
+const useStyles = makeStyles(theme => ({
+  form: {
+    width: '100%',
+    marginTop: theme.spacing(1)
+  },
+  logo: {
+    maxWidth: '70%',
+    display: 'block',
+    margin: 'auto',
+    padding: theme.spacing(1)
+  },
+  helpIcon: {
+    fontSize: 'medium',
+    cursor: 'default'
+  },
+  runButton: {
+    borderRadius: theme.spacing(2)
+  },
+  actionSection: {
+    padding: theme.spacing(2)
+  },
+  progressBar: {
+    padding: theme.spacing(2)
+  },
+  errorSection: {
+    padding: theme.spacing(2),
+    borderRadius: theme.spacing(1)
+  }
+}))
+
+export default function RunOptionsForm(props) {
+  const classes = useStyles()
+
+  const [options, setOptions] = useState({
+    url: 'https://google.com',
+    connections: 10,
+    pipelining: 1,
+    duration: 10,
+    method: 'GET'
+  })
```

Comment:
nit: let's extract the initial value in a constant outside the component body, say DEFAULT_OPTIONS